packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -395,17 +395,40 @@ export function getStylesDeclarations(
 	);
 
 	/*
-	 * Set background defaults.
-	 * Applies to all background styles except the top-level site background.
+	 * Preprocess background image values.
+	 * 1. Sets default values for blocks (not root).
+	 * 2. .
```

Comment:
Should this be something like the following?

```suggestion
	 * 2. Resolve dynamic values before they are compiled by the style engine.
```